### PR TITLE
feat: updates docs to call out os pre-reqs explicitly

### PR DESCRIPTION
**Pull request by eaddingtonwhite**

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -1,13 +1,31 @@
...
 _Read this in other languages_: [日本語](README.ja.md)
-
 <br>
 
-## Running the Example Using Pipenv
+## Prereqs
 
 - [Python 3.7 or above is required](https://www.python.org/downloads/)
 - A Momento Auth Token is required, you can generate one using the [Momento CLI](https://github.com/momentohq/momento-cli)
+- Developer librarys (gcc/python dev headers) installed on machine you intend to run on
```
Comment (pgautier404) [inline python/README.md:10]:
```suggestion
- Developer libraries (gcc/python dev headers) installed on machine you intend to run on
```

**Review by ksmotiv8 — APPROVED** [review]
Looks good

**Comment by malandis** [comment]
Looks good. One thing regarding this and other READMEs: our translations are (or will be) out of sync with the English ones. Need to address README language drift.

**Comment by eaddingtonwhite** [comment]
@malandis good call let me get other ones updated

**Comment by eaddingtonwhite** [comment]
Actually @malandis its more out of date then just this pr. Im going to get translator to come back in and get fully up to date in follow up pr. Good call out though we need to address this longer term have better process here.

**Pull request merged**